subsys/zbus/Kconfig

```
@@ -79,11 +79,14 @@ config ZBUS_ASSERT_MOCK
 
 
 config HEAP_MEM_POOL_ADD_SIZE_ZBUS
-	int
+	int "Zbus message subscriber heap pool size."
```

The heap is not only for message subscribers. 

```suggestion
	int "Zbus requested heap pool size."
```

Curiosity only. No other subsystem in Zephyr has a prompt for adding heap size. All of them are defined internally. I see value in it, allowing the user to change that in the application configuration, since the applications will differ.